docs/src/views/Accessibility.tsx

```diff
@@ -10,56 +12,56 @@ export default () => (
     <Header as="h2" content="Content" />
     <ul>
       <li>
-        {link('Goals of accessibility', '/accessibility#goals-of-accessibility')}
+        {link('Goals of accessibility', 'accessibility#goals-of-accessibility')}
         <ul>
-          <li>{link('Out of scope', '/accessibility#out-of-scope')}</li>
+          <li>{link('Out of scope', 'accessibility#out-of-scope')}</li>
         </ul>
       </li>
       <li>
-        {link('Making an app / page accessible', '/accessibility#making-an-app-page-accessible')}
+        {link('Making an app / page accessible', 'accessibility#making-an-app-page-accessible')}
         <ul>
-          <li>{link('Semantic HTML', '/accessibility#semantic-html')}</li>
-          <li>{link('Design Considerations', '/accessibility#design-considerations')}</li>
+          <li>{link('Semantic HTML', 'accessibility#semantic-html')}</li>
+          <li>{link('Design Considerations', 'accessibility#design-considerations')}</li>
         </ul>
       </li>
       <li>
-        {link('Keyboard Navigation', '/accessibility#keyboard-navigation')}
+        {link('Keyboard Navigation', 'accessibility#keyboard-navigation')}
```

Comment:
given that those are the links to the same page, suggest to use just id selector as their content - this would solve all the maintainability headache:

```jsx
{link('Keyboard Navigation', '#keyboard-navigation')}
```

Reply:
I've tried that approach, unfortunately, it does not work that way. That query part is not added to the end but rewrites the path